scripts/Engineering/EnginX.py

```diff
@@ -14,11 +14,16 @@
 import Engineering.EnggUtils as Utils
 import mantid.simpleapi as simple
 
+NORTH_BANK_CAL = "EnginX_NorthBank.cal"
+SOUTH_BANK_CAL = "EnginX_SouthBank.cal"
```

Comment:
Maybe this is for another PR - but I have had a go at using the algorithm CreateGroupingWorkspaceand eventually figured out how to get it working! So instead of keeping these cal files (one for each bank) we can instead get a grouping workspace to focus e.g. for the north bank
```
Load(Filename=r'\\isis.cclrc.ac.uk\inst$\ndxenginx\instrument\data\cycle_12_4\ENGINX00193749.nxs', OutputWorkspace='193749')
CreateGroupingWorkspace(InputWorkspace='193749', GroupNames='NorthBank', OutputWorkspace='north_grp')
DiffractionFocussing(InputWorkspace='193749', OutputWorkspace='foc', GroupingWorkspace='north_grp')

```

Reply:
Thanks for investigating that, I'll make that part of the custom cal file PR